tests/setup.js

```diff
@@ -3,24 +3,36 @@
 const { getHttpOperationsFromSpec }  = require('@stoplight/prism-cli/dist/operations');
 const { createClientFromOperations } = require('@stoplight/prism-http/dist/client');
 const { URL }                        = require('url');
+const btoa                           = require('btoa');
```

Comment:
@hilary can you add prettier to this repo and be done with this assignment/alignment stuff?

Reply:
Yup, will do.